src/MathJaxRenderer.jl

```diff
@@ -126,29 +126,31 @@ function svg_converter(
     return IOBuffer(extract_svg(str))
 end
 
+
 function converter(
     f::Math,
     m::SUPPORTED_MIMES;
     dpi_x = 90,
     dpi_y = 90,
     x_zoom = 1.0,
     y_zoom = 1.0,
-    zoom = 1.0,
     keep_aspect_ratio = false,
     background_color = :white,
     unlimited = false,
+    rsvg_convert_bin = "",
     kws...,
 )
     Librsvg_jll.rsvg_convert() do bin
-        ext = extension(m)
+        if !isempty(rsvg_convert_bin)
+            bin = rsvg_convert_bin
```

Comment:
Probably worth checking whether the provided binary actually is `rsvg_convert` rather than just calling it regardless of what the user provided.

Reply:
I implemented a check in 81f1cd4bc43cc1c0cfd9dc4c48c822e24c66b684, I like having the ability to call another command, however. Feel free to change it :)

Reply:
Yes, that sounds reasonable.